cmd/access/node_builder/access_node_builder.go

```diff
@@ -2123,7 +2140,10 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 				node.State,
 				node.Storage.Receipts,
 				execNodeSelector,
-				optimistic_sync.DefaultCriteria,
```

Comment:
`DefaultCriteria` is also passed to the backend params as the `OperatorCriteria` field. I don't think we need that field anymore. can you remove it from the params struct?